extensions/ANGLE/EGL_ANGLE_sync_control_rate.txt

```diff
@@ -0,0 +1,110 @@
+﻿Name
+
+   ANGLE_sync_control_rate
+
+Name Strings
+
+   EGL_ANGLE_sync_control_rate
+
+Contact
+
+   Jonah Ryan-Davis, Google (jonahr 'at' google.com)
+
+Status
+
+   Draft.
+
+Version
+
+   Version 1, 2020-03-24
+
+   Based on GLX_OML_sync_control Revision 6.0
+
+Number
+
+   ???
```

Comment:
This should be 142 (or whatever's next in `registry.tcl`, if something else gets merged first).